services/node_create.proto

```diff
@@ -125,4 +127,14 @@ message NodeCreateTransactionBody {
      * This field is OPTIONAL.
      */
     bytes grpc_certificate_hash = 6;
+
+    /**
+    * An administrative key for the node address book entry.
+    * <p>
+    * This key MUST sign this transaction.<br/>
+    * This key MUST sign each transaction to update this node.<br/>
+    * This field is REQUIRED and MUST NOT be set to an empty `KeyList`.
+    * This key MUST be valid.
```

Comment:
We could probably combine like this. Or any other way @jsync-swirlds  suggests

```suggestion
    /**
    * The administrative key for the node address book entry.
    * <p>
    * This field is REQUIRED. It MUST be a valid key and MUST NOT be set to an empty `KeyList`.<br/>
    * This key MUST sign this transaction and each transaction to update this node.<br/>
```

Reply:
Updated.

Reply:
A couple concerns:
1. Line limit is 80 characters, this makes the lines longer (we could split the lines, but this suggestion did not).
1. The change is much less of a specification.  Specification language should be a bit more staccato than typical speech.
   * generally, replace "will" with "SHALL" to specify expectable behavior.
   * generally, each sentence is best as a separate line.
   * generally, behavior for different transactions should be separate lines.

I have a document with all the recommendations for specification text, but it's not final. I can provide that draft document separately if you're interested.

Reply:
Updated